README.md

```diff
@@ -444,11 +444,11 @@ Besides that, `swag` also accepts aliases for some MIME Types as follows:
 | annotation | description | parameters | example |
 |------------|-------------|------------|---------|
 | securitydefinitions.basic  | [Basic](https://swagger.io/docs/specification/2-0/authentication/basic-authentication/) auth.  |                                   | // @securityDefinitions.basic BasicAuth                      |
-| securitydefinitions.apikey | [API key](https://swagger.io/docs/specification/2-0/authentication/api-keys/) auth.            | in, name                          | // @securityDefinitions.apikey ApiKeyAuth                    |
-| securitydefinitions.oauth2.application  | [OAuth2 application](https://swagger.io/docs/specification/authentication/oauth2/) auth.       | tokenUrl, scope                   | // @securitydefinitions.oauth2.application OAuth2Application |
-| securitydefinitions.oauth2.implicit     | [OAuth2 implicit](https://swagger.io/docs/specification/authentication/oauth2/) auth.          | authorizationUrl, scope           | // @securitydefinitions.oauth2.implicit OAuth2Implicit       |
-| securitydefinitions.oauth2.password     | [OAuth2 password](https://swagger.io/docs/specification/authentication/oauth2/) auth.          | tokenUrl, scope                   | // @securitydefinitions.oauth2.password OAuth2Password       |
-| securitydefinitions.oauth2.accessCode   | [OAuth2 access code](https://swagger.io/docs/specification/authentication/oauth2/) auth.       | tokenUrl, authorizationUrl, scope | // @securitydefinitions.oauth2.accessCode OAuth2AccessCode   |
+| securitydefinitions.apikey | [API key](https://swagger.io/docs/specification/2-0/authentication/api-keys/) auth.            | in, name, sec.def.description                          | // @securityDefinitions.apikey ApiKeyAuth                    |
+| securitydefinitions.oauth2.application  | [OAuth2 application](https://swagger.io/docs/specification/authentication/oauth2/) auth.       | tokenUrl, scope, sec.def.description                   | // @securitydefinitions.oauth2.application OAuth2Application |
+| securitydefinitions.oauth2.implicit     | [OAuth2 implicit](https://swagger.io/docs/specification/authentication/oauth2/) auth.          | authorizationUrl, scope, sec.def.description           | // @securitydefinitions.oauth2.implicit OAuth2Implicit       |
+| securitydefinitions.oauth2.password     | [OAuth2 password](https://swagger.io/docs/specification/authentication/oauth2/) auth.          | tokenUrl, scope, sec.def.description                   | // @securitydefinitions.oauth2.password OAuth2Password       |
+| securitydefinitions.oauth2.accessCode   | [OAuth2 access code](https://swagger.io/docs/specification/authentication/oauth2/) auth.       | tokenUrl, authorizationUrl, scope, sec.def.description | // @securitydefinitions.oauth2.accessCode OAuth2AccessCode   |
```

Comment:
Please update sec.def.description with "description" for the parameter column and add also the "description" feature in the example column.

Reply:
I added the example on the table below where the `parameter annotation | example` lives as I cannot do line breaks on the example cell of the mentioned table. Hope that is ok.

Reply:
@ubogdan , as `sec.def.description` is a "line" parameter such as `header`, `in`, `scopes` etc; I had to name it in the mentioned way because if I named it `description` it will override (or take) the description at the general API info level.

Reply:
I can of course change it to `x-description` as well as an alternative :)

Reply:
Then @securitydefinitions.description should be more appropiate?

Reply:
I can do that :)

Reply:
You made it @security.definition.description instead of @description ...

Reply:
Sorry, did not see your edited comment 😅 
I will try to make it work with just `description` but as I said before, when I tried it overlapped with the `@description` that you put to the general API info. I will debug and see if I can solve that.

Reply:
The current way it works is the following, given these lines:

```
// @description this API is a demo
// @securitydefinitions.oauth2.application  OAuth2Application
// @description                             Perform OAUTH2 at application level
```

The code will:

- Range every comment
- For the first line it will set the API description to `this API is a demo`
- For the second line it will range all comments starting from `comments[i+1:]` in other words skipping the current. This will effectively go through the `@description` and set it correctly.
- For the third line it will set the API description (and override) to `Perform OAUTH2 at application level`

The issue is that the `parseSecAttr` function is iterating all the lines below until it finds another `@securitydefinitions.`. But then, the lines that `parseSecAttr` iterated, will be iterated back again by the main for buckle.

This is why I changed the name to something different. I can potentially see how to make the main buckle not go through the lines that the security definition already did, but that might introduce lots of changes.

Reply:
@ubogdan check it now, with the changes I have done it is working correctly and the main buckle does not re-iterate the lines that the security definitions already iterated :)